app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ConversationsController < ApplicationController
+
+  def show
+    @conversation = find_convo
+  end
+
+  def create
+
+    @conversation = Conversation.new(conversation_params)
+    @conversation.messages.first.user = current_user
+    if @conversation.save
+      if request.xhr?
+        render partial: "topics/conversation", locals: {conversation: @conversation}
+      else
+        redirect_to @conversation
+      end
+    else
+      if request.xhr?
+        render plain: "Insufficient gerbils", :status => 422
+      else
+        @errors = @conversation.errors.full_messages
```

Comment:
Generally prefer the flash hash as a place to store errors for both renders and redirects in Rails.